terraform/fullnodes.tf

```diff
@@ -38,14 +38,6 @@ resource "aws_instance" "fullnode" {
 
 }
 
-data "template_file" "fullnode_config" {
-  template = file("templates/fullnode.yaml")
-
-  vars = {
```

Comment:
delete the base config too? also in the validator/fullnodes.json

Reply:
that defines some properties, I think we keep that, I'm gonna use this push result to let me know what I'm doing wrong :p